packages/gatsby-plugin-page-creator/src/gatsby-node.js

```diff
@@ -48,48 +49,56 @@ exports.createPagesStatefully = async (
     )
   }
 
-  const pagesDirectory = systemPath.posix.join(pagesPath)
-  const pagesGlob = `${pagesDirectory}/**/*.{${exts}}`
+  const pagesDirectory = systemPath.resolve(process.cwd(), pagesPath)
```

Comment:
should we move to posix here?

Reply:
Since `pagesDirectory` is only being used is to form the component path in createPages, it should be consistent and use the same OS path implementation as the previous two comments.

Reply:
Since `process.cwd()` and `pagesPath` are system-dependent paths, I suspect `systemPath.resolve` is right.